packages/ag-charts-community/src/chart/toolbar/toolbar.ts

```diff
@@ -512,6 +557,12 @@ export class Toolbar extends BaseModuleInstance implements ModuleInstance {
                 this.setButtonActive(button, active);
             }
         }
+
+        if (this[group].draggable) {
+            this.ctx.toolbarManager.toggleButton(group, 'drag', {
+                enabled,
+            });
+        }
```

Comment:
Calling from `toolbar` out to the `toolbarManager` to trigger a listener inside `toolbar` is a bit circuitous.

What do you need from the `onButtonToggled()` listener to happen? Other than the `setButtonActive()` that's already called a few lines above?

Reply:
I think I left that in there by accident :)